src/lib.rs

```diff
@@ -305,7 +305,7 @@ pub mod kv;
 
 // The LOGGER static holds a pointer to the global logger. It is protected by
 // the STATE static which determines whether LOGGER has been initialized yet.
-static mut LOGGER: &'static Log = &NopLogger;
+static mut LOGGER: &dyn Log = &NopLogger;
```

Comment:
These changes will increase the minimum supported Rust version.

Reply:
Ok, removing these chances.